wdio.conf.ts

```diff
@@ -0,0 +1,309 @@
+/// <reference types="wdio-electron-service" />
+import type { Options } from '@wdio/types';
+
+export const config: Options.Testrunner = {
+	//
+	// ====================
+	// Runner Configuration
+	// ====================
+	// WebdriverIO supports running e2e tests as well as unit and component tests.
+	runner: 'local',
+	autoCompileOpts: {
+		autoCompile: true,
+		tsNodeOpts: {
+			project: './tsconfig.json',
+			transpileOnly: true,
+		},
+	},
+
+	//
+	// ==================
+	// Specify Test Files
+	// ==================
+	// Define which test specs should run. The pattern is relative to the directory
+	// of the configuration file being run.
+	//
+	// The specs are defined as an array of spec files (optionally using wildcards
+	// that will be expanded). The test for each spec file will be run in a separate
+	// worker process. In order to have a group of spec files run in the same worker
+	// process simply enclose them in an array within the specs array.
+	//
+	// The path of the spec files will be resolved relative from the directory of
+	// of the config file unless it's absolute.
+	//
+	specs: ['./tests/**/*.spec.ts'],
+	// Patterns to exclude.
+	exclude: [
+		// 'path/to/excluded/files'
```

Comment:
Can you please use this directive to exclude the test files instead of moving them around?